language/ir-testsuite/tests/move/comments/single_line_comment_line.mvir

```diff
@@ -1,3 +1,3 @@
 main() {
-  return; // This is a comment
+  return; // This is a comment. Comments may contain Unicode 😇
```

Comment:
We intentionally do not include unicode but we may relax this in the future. This was due to "security" issues but... maybe it is time to discuss this again

Reply:
I decided not to mess with this aspect of the IR for now. I did add a comment to the `verify_string` function in ir-to-bytecode that mentions an alternative. It does seem more fool-proof to me to greatly restrict the input character set, and to check that up front before any lexing occurs.